src/ucp/core/ucp_context.c

```diff
@@ -159,8 +159,8 @@ static ucp_tl_alias_t ucp_tl_aliases[] = {
   { "sm",    { "mm", "knem", "sysv", "posix", "cma", "xpmem", NULL } },
   { "shm",   { "mm", "knem", "sysv", "posix", "cma", "xpmem", NULL } },
   { "ib",    { "rc", "ud", "rc_mlx5", "ud_mlx5", NULL } },
-  { "rc",    { "rc", "ud", NULL } },
-  { "rc_x",  { "rc_mlx5", "ud_mlx5", NULL } },
+  { "rc",    { "rc", "ud:aux", NULL } },
+  { "rc_x",  { "rc_mlx5", "ud_mlx5:aux", NULL } },
```

Comment:
need to also set ugni_udt:aux, right @MattBBaker ?

Reply:
do we need to introduce new alias, which would imply that udt is for wireup only? Now we have just a single `ugni` alias with all possible transports (like `ib`)

Reply:
udt is for wireup only, so it fits perfectly for being aux. If this pulls udt out of ugni and makes ugni just rdma and smsg, that is fine.

Reply:
ok